py-scripts/lf_mixed_traffic.py

```diff
@@ -2015,7 +2015,8 @@ def generate_all_report(self):
                     self.lf_report_mt.set_csv_filename(graph_png)
                     self.lf_report_mt.move_csv_file()
                     self.lf_report_mt.build_graph()
-                    qos_obj.generate_individual_graph(self.res, self.lf_report_mt, qos_obj.connections_download_avg, qos_obj.connections_upload_avg, qos_obj.avg_drop_a, qos_obj.avg_drop_b,self.total_floors,multicast_exists=True if "5" in self.tests and self.get_live_view else False)
+                    qos_obj.generate_individual_graph(self.res, self.lf_report_mt, qos_obj.connections_download_avg, qos_obj.connections_upload_avg, qos_obj.avg_drop_a,
+                                                      qos_obj.avg_drop_b, self.total_floors, multicast_exists=True if "5" in self.tests and self.get_live_view else False)
```

Comment:
This is fairly verbose, but okay. In the future, would like to see any more complex function calls with arguments separated on newlines.